impl/src/main/java/jakarta/faces/event/PhaseId.java

```diff
@@ -248,6 +239,13 @@ public static PhaseId phaseIdValueOf(String phase) {
      * List of valid {@link PhaseId} instances, in ascending order of their ordinal value.
      * </p>
      */
-    public static final List<PhaseId> VALUES = Collections.unmodifiableList(Arrays.asList(values));
+    public static final List<PhaseId> VALUES = List.of(values);
+
+    /**
+     * <p>
+     * Valid {@link PhaseId} instances, mapped by their uppercase name
+     * </p>
+     */
+    public static final Map<String,PhaseId> VALUES_BY_NAME = unmodifiableMap(Stream.of(values).collect(toMap( phase -> phase.getName().toUpperCase() , identity())));
```

Comment:
This is a new field in the API. Please create a ticket in https://github.com/jakartaee/faces/issues referencing this PR. And you also you need to make 1st paragraph of javadoc a `<p class="changed_added_4_1">` and add a `@since 4.1` to the end. See also e.g. https://github.com/eclipse-ee4j/mojarra/pull/5187/commits/5e61f0d9b419e419d413c4509c8797809f54aac6 for an example.

Reply:
Or change to `private` if you didn't intend to expose this in the public API.

Reply:
changed to private for now...

It's better to transform `PhaseId` to an `enum`